android/tangram/jni/platform_android.cpp

```diff
@@ -302,7 +302,10 @@ void featureSelectionCallback(JNIEnv* jniEnv, const std::vector<Tangram::TouchIt
     jresult = reinterpret_cast<jlong>(new std::shared_ptr<Tangram::Properties >(items[0].properties));
     jobject object = jniEnv->NewObject(propertiesClass, propertiesConstructorMID, jresult, true);
 
-    jniEnv->CallVoidMethod(tangramInstance, featureSelectionCbMID, object);
+    jdoubleArray itemCenter = jniEnv->NewDoubleArray(2);
```

Comment:
I am curious too - it's the "New" call that makes me think this needs to be released, the "Region" call should be fine as it is.